simple-service-webapp-rest/src/main/java/com/example/rest/AuthenticationEndpoint.java

```diff
@@ -38,8 +38,10 @@ private void authenticate(String username, String password) throws Exception {
         if(username.equals("abcde"))
             if(password.equals("12345"))
                 return;
-        else
-            throw new Exception();
+        else {
+                System.out.println("ham");
+                throw new Exception();
```

Comment:
Token should be generated and random, use UUID.randomUUID()
Token should be an object, which has the token string and also a Date for when it was created.
Filter should check if the token is valid, and also check if it has timed out. the timeout should be hardcoded, for ex. 1 minute for easy testing. If the token expired, the server should return 401